ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -206,6 +206,14 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				Description: "The operating system of the workers in the default worker pool.",
 			},
 
+			"secondary_storage": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
```

Comment:
is this computed also? If user doesn't create any secondary storage does API set back any default value